boards/beagle/beagleconnect_freedom/board_antenna.c

```diff
@@ -1,116 +1,119 @@
-/* SPDX-License-Identifier: Apache-2.0
- *
```

Comment:
I guess this seems right compared to existing source in Zephyr, but differs from SPDX examples. What made you decide to change it?

Reply:
Most other actual files including launchxl board_antenna.c seem to follow it.

Reply:
Can you please move all formatting and non-functional changes (copyright, etc.) in a separate preparative commit as proposed by @jadonk ? I'm not against formatting changes if they improve readability but these make it unnecessarily hard to review the actual functional change for correctness.

I also saw that some of your changes seem to introduce new differences between https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/ti/cc1352p1_launchxl/board_antenna.c and this file. As it is your intention to make the files more similar this should be fixed.

If you make improvements here (like improved formatting or additional comments) then please also make them there (in the same commit).

I recommend you make a diff between the two final files and ensure that only required functional changes differ between them.